rites/roman1969/src/calendars/countries/brazil/index.ts

```diff
@@ -25,6 +25,12 @@ export class Brazil extends CalendarDef {
       ],
     },
 
+    ephrem_the_syrian_deacon: {
+      // In Brazil, St Ephrem is transferred to June 8
+      // because St Joseph de Anchieta is celebrated on June 9
```

Comment:
```suggestion
      // because St Joseph de Anchieta is celebrated on June 9
      // src: `https://arquidiocesemilitar.org.br/wp-content/uploads/2025/01/DIRETORIO-LITURGICO-OMB-2025-1_compressed.pdf#page=79`
```